### ISSUE TITLE
Add help to CLI

### ISSUE THREAD

**Issue by christian-reichart** [christian-reichart/svg-chameleon#11]
Help command should list all available CLI options with a short description.

We have two options here:

1.
`svg-chameleon help` to list the help output and run the create script with `svg-chameleon` (this is how it is currently working)

2. 
`svg-chameleon` or `svg-chameleon help` to list the help output and `svg-chameleon create` to run the create script.

Not sure here, but I'm leaning towards option 2. 
Because the create script actually creates files it should not be run by accident. Many other tools list their help commands when you just type in the name, so it is expected of many users.

**Comment by TheJotob** [comment]
Yep. I would also like option 2 more. I often just run a command without any arguments to get the help document. Because there are many different ways to get the help document.
Also separating commands helps to keep the code cleaner :)

**Comment by leChuckles** [comment]
@christian-reichart 
could you please add me as a developer on the project? I completed this issue (option 2) and would like to create a PR for you :)

**Comment by christian-reichart** [comment]
@leChuckles nice! You can just create a PR from your fork, or what do you mean?

https://opensource.com/article/19/7/create-pull-request-github

**Comment by leChuckles** [comment]
@christian-reichart https://github.com/christian-reichart/svg-chameleon/pull/22
